internal-registry/che-incubator/command-line-terminal-dev/nightly/meta.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+publisher: che-incubator
+name: command-line-terminal
+version: nightly
+type: Che Editor
+displayName: Command Line Terminal
+title: Command Line Terminal
+description: Command Line Terminal provides the ability to start a terminal inside
```

Comment:
I think it worths mentioning here why dev version exist and how it's supposed to be used - for openshift local bridge...

Reply:
Updated description to mention that it doesn't run with TLS enabled. I think there are uses beyond running bridge locally, as this version is required for all `routingClass`es other than `cluster-tls`.